mava/components/training/model_updating.py

```diff
@@ -93,6 +95,18 @@ def on_training_utility_fns(self, trainer: SystemTrainer) -> None:
             None.
         """
 
+        # Initilaise running statisitics here
+
+        if self.config.normalize_target_values:
+            trainer.store.running_stats_fn = compute_running_mean_var_count
+            running_stats = jnp.array([0, 0, 1e-4])
```

Comment:
Looks we should take `running_stats = jnp.array([0, 0, 1e-4])` out the if statement since it applies in both conditions.

Reply:
In the other case there is a slight difference we have 1 instead of 0 for the variance. 
`if self.config.normalize_target_values:
            trainer.store.running_stats_fn = compute_running_mean_var_count
            running_stats = jnp.array([0, 0, 1e-4])
        else:
            running_stats = jnp.array([0, 1, 1e-4])`